priv/repo/migrations/20241211135758_certifications.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Orbit.Repo.Migrations.CertificationsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:certifications) do
+      add(:badge, :string)
+      # Enum
+      add(:type, :integer)
+      # Enum
+      add(:rail_line, :rail_line)
+      add(:expires, :date)
```

Comment:
All or most of these fields should probably get `null: false`.

Reply:
Agreed, fixed https://github.com/mbta/orbit/pull/203/files#diff-248c4b179027894763ea37f20c43c9e2ac96ea5b22619c4e898777d809a5c8b5R6